spec/cobrato/resources/charge_spec.rb

```diff
@@ -81,9 +81,9 @@
   describe "#create" do
     it "creates a charge" do
       VCR.use_cassette("charges/create/success") do
-        charge_account = subject.create(params)
-        expect(charge_account).to be_a(entity_klass)
-        expect(charge_account.our_number).to eq(params['our_number'])
+        charge = subject.create(params)
+        expect(charge).to be_a(entity_klass)
+        expect(charge.our_number).to eq(params['our_number'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.